code_folder/A1_csv_to_json.py

```diff
@@ -0,0 +1,356 @@
+import pandas as pd
+import os
+import json
+from pathlib import Path
+import logging
+
+"""
+How to use:
+all default input csv"s are stored in "/mvs_eland/inputs/elements/default_csv".
+These csvs are not to be changed! All csvs that you need in order to set up 
+your energy system should be stored into "/mvs_eland/inputs/elements/csv". Here
+you can change parameters and add components. The given parameters need to be 
+maintained. 
+Storage: The "energyStorage.csv" contains information about all storages. 
+For each storage there needs to be another file named exactly as the according 
+storage-column in "energyStorage.csv", usually this is "storage_01", 
+"storage_02" etc. Please stick to this convention.
+After all the function "infer_resources()" reads all csv that are stored in 
+the folder "/mvs_eland/inputs/elements/csv" and creates one json input file 
+for mvs
+"""
+
+
+def infer_resources():
+
+    """
+    Method looks at all csv-files in "mvs_eland/inputs/elements/csv",
+    converts them into json files and joins them together into the file
+    "mvs_eland/inputs/working_example2.json". When reading the csv files it is
+    checked, weather all required parameters for each component are given.
+    Missing parameters will return a message.
+
+    Parameters
+    -------------
+    None
+
+    Returns
+    --------------
+    None
+        the json file "working_example2.json" is saved
+
+    """
+    input_directory = os.path.join(
+        Path(os.path.dirname(__file__)).parent, "inputs/elements/"
+    )
+    output_filename = "working_example2.json"
```

Comment:
Wouldn't it be good to have this variable as argument of the function with default value in case we need to change the name of the output_filename in the future?